mne/viz/ica.py

```diff
@@ -83,10 +84,11 @@ def plot_ica_sources(ica, inst, picks=None, exclude=None, start=None,
     from ..evoked import Evoked
     from ..epochs import BaseEpochs
 
-    if exclude is None:
-        exclude = ica.exclude
-    elif len(ica.exclude) > 0:
-        exclude = np.union1d(ica.exclude, exclude)
+    if exclude is not None:
```

Comment:
It's a bit more than this. A user with 0.19 could pass `exclude=None`, get no warning, and in 0.20 their code will error.

To avoid this you can set `exclude='deprecated'` as the default, which behaves like `None` but does not warn.

Reply:
good point. pushed a fix.